siege_utilities/geo/boundary_providers.py

```
@@ -227,3 +228,141 @@ def resolve_boundary_provider(country: str = 'US', **kwargs: Any) -> BoundaryPro
+    * ``'state_senate'`` — enacted upper-chamber plans
+    * ``'state_house'`` — enacted lower-chamber plans
+
+    Usage::
+
+        from siege_utilities.geo.boundary_providers import RDHProvider
+
+        provider = RDHProvider(username="you@example.com", password="secret")
+        gdf = provider.get_boundary('precinct', identifier='TX')
+    """
+
+    _LEVELS = ['precinct', 'congress', 'state_senate', 'state_house']
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)